### PR TITLE
Use new sum method on spans to better avoid collisions

### PR DESCRIPTION
**Pull request by Foretack**

### CODE REVIEW

File: MiniTwitch.Irc.Test/SpanSumTests.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        {
+            if (kvp.Key != MSum(kvp.Value))
+            {
+                Assert.Fail($"Tag \"{kvp.Value}\" does not match expected sum {kvp.Key}. Got {MSum(kvp.Value)} instead.");
+            }
+        }
+    }
+
+
+    private static int MSum(string s)
```
Comment (jupjohn) [inline MiniTwitch.Irc.Test/SpanSumTests.cs:112]: This should really be a wrapped call to your implementation from `SpanExtensions`, you're only testing your test implementation of `MSum` here

File: MiniTwitch.PubSub.Test/SpanSumTests.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        foreach (var kvp in _types)
+        {
+            if (kvp.Key != MSum(kvp.Value))
+            {
+                Assert.Fail($"Type \"{kvp.Value}\" does not match expected sum {kvp.Key}. Got {MSum(kvp.Value)} instead.");
+            }
+        }
+    }
+
+    private static int MSum(string s)
```
Comment (jupjohn) [inline MiniTwitch.PubSub.Test/SpanSumTests.cs:115]: Same as my other test class comment: you're not testing your implementation.

**Review by jupjohn — APPROVED** [review]
LGTM :+1:

**Comment by Foretack** [comment]
Tested, seems good

**Pull request merged**